app/views/amr_heatmap/visualization.html.erb

```diff
@@ -0,0 +1,10 @@
+<div id="AmrHeatmapVis_page">
+</div>
+
+<%= javascript_tag do %>
+  react_component(
+    'AMRHeatmapVis',
+    JSON.parse('<%= raw escape_json(@samples) %>'),
+    'AmrHeatmapVis_page'
```

Comment:
Nit: Use `page_content` which already exists instead of creating another div `AmrHeatmapVis_page`.